Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
+      - >
+        go run github.com/air-verse/air@v1.61.1 \
+        --build.cmd "go build -o tmp/bin/main" --build.bin "tmp/bin/main" --build.delay "100" \
```

Comment:
For commands like `go run` and `go test`, ensure that any sensitive data needed (like API keys or tokens) is properly managed to avoid leaks, especially in logs. This is especially important with `GITHUB_TOKEN` and `OPENAI_API_KEY` that are mentioned as required environment variables. Consider whether these should be masked in any output.